acqa-repo1-aws-tf12-part1.tf

```diff
@@ -391,20 +413,20 @@ resource "aws_subnet" "acqa-test-albsubnet1" {
   cidr_block              = "10.0.44.0/24"
   map_public_ip_on_launch = true
   tags = {
-    Name = format("%s-albsubnet1", var.acqaPrefix)
+    Name         = format("%s-albsubnet1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "ACQA"
+    Owner        = "ACQA"
   }
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure deletion protection is enabled for AWS LB (Load Balancer)**
 
**Vulnerability**
AWS Classic load balancer has deletion protection disabled. This may lead to accidental deletion of the load balancer.
 
**Remediation**
Enable deletion protection for classic load balancer. This will prevent accidental deletion of the resource.

source: accurics-